pkg/rkeworker/docker_linux.go

```diff
@@ -0,0 +1,162 @@
+package rkeworker
```

Comment:
whats the purpose of this file?

Reply:
os dependent versions: one for linux (this one) and one for windows (pkg/rkeworker/docker_windows.go)